### ISSUE TITLE
Remove recommendation to have oceansites_version attribute in the file.

### ISSUE THREAD

**Issue by dpsnowden** [oceansites/dmt#11]
Change data format guide to remove the oceansites version global attribute and use the CF recommended `Conventions: CF-1.6 Oceansites-1.3`.

**Comment by ngalbraith** [comment]
I believe this is used by the GDAC software. I've asked Jing and Thierry if they can use the Conventions attribute instead.

**Comment by dpsnowden** [comment]
Email response from Jing

Nan,

This issue was not raised by Derrick first. I believed he just recalled the issue I discussed during my presentation at the OceanSITES meeting this week. I can identify ~28000 data files having version OceanSITES 1.2 or 1.3 based on the Convention attribute. But there are 3000~4000 data files for which their versions can not be recognized from the Convention attribute. 

Someone suggested we should look at the format_version attribute. Earlier data files might use that attribute to document their version. Because we know that Convention attribute should be used in the future, so we discussed the recommendation to drop the format_version in Data Reference Guide 1.4. I did not notice any objection during our discussion. Also there are no GDAC processing programs that depend on the format_version attribute to my knowledge, dropping it should not cause problems at GDAC side, it would make the reference guide clean.

Thanks,

-Jing
